src/main/java/com/github/cameltooling/lsp/internal/completion/CompletionResolverUtils.java

```diff
@@ -16,18 +16,31 @@
  */
 package com.github.cameltooling.lsp.internal.completion;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.camel.parser.helper.CamelXmlHelper;
 import org.eclipse.lsp4j.CompletionItem;
 import org.eclipse.lsp4j.Position;
 import org.eclipse.lsp4j.Range;
+import org.eclipse.lsp4j.TextDocumentItem;
 import org.eclipse.lsp4j.TextEdit;
+import org.w3c.dom.Node;
 
+import com.github.cameltooling.lsp.internal.instancemodel.CamelURIInstance;
 import com.github.cameltooling.lsp.internal.instancemodel.CamelUriElementInstance;
+import com.github.cameltooling.lsp.internal.instancemodel.PathParamURIInstance;
+import com.github.cameltooling.lsp.internal.parser.ParserXMLFileHelper;
 
 /**
  * @author lheinema
  */
 public class CompletionResolverUtils {
 	
+	private static final List<String> POSSIBLE_DIRECT_REFERENCE = Arrays.asList("direct","direct-vm", "seda", "vm", "amqp", "jms", "activemq", "broker");
```

Comment:
activemq seems to not be part of camel-core either 

http://camel.apache.org/activemq.html

![image](https://user-images.githubusercontent.com/1105127/46077135-1a919500-c190-11e8-89d0-30057e7f3b9f.png)